paper-fab.html

```diff
@@ -60,29 +60,30 @@
   <template strip-whitespace>
     <style include="paper-material-shared-styles">
       :host {
-        display: inline-block;
-        position: relative;
-        outline: none;
-        -moz-user-select: none;
-        -ms-user-select: none;
-        -webkit-user-select: none;
-        -webkit-tap-highlight-color: rgba(0,0,0,0);
-        user-select: none;
-        cursor: pointer;
+        @apply(--layout-vertical);
+        @apply(--layout-center-center);
 
-        box-sizing: border-box;
-        min-width: 0;
-        width: 56px;
-        height: 56px;
         background: var(--paper-fab-background, --accent-color);
-        color: var(--text-primary-color);
         border-radius: 50%;
+        box-sizing: border-box;
+        color: var(--text-primary-color);
+        cursor: pointer;
+        height: 56px;
+        min-width: 0;
+        outline: none;
         padding: 16px;
-
+        position: relative;
+          -moz-user-select: none;
+          -ms-user-select: none;
+          -webkit-user-select: none;
+        user-select: none;
+        width: 56px;
         z-index: 0;
 
-        @apply(--layout-vertical);
-        @apply(--layout-center-center);
+        /* NOTE: Both values are needed, since some phones require the value `transparent`. */
+        -webkit-tap-highlight-color: rgba(0,0,0,0);
+        -webkit-tap-highlight-color: transparent;
```

Comment:
IIRC reason for this hack is as follows: Safari iOS does not support color names or hex values for `-webkit-tap-highlight-color` when you want remove tap highlights. This may have been addressed in more recent betas.